### PR TITLE
Wording

### PR DESCRIPTION
**Pull request by yamboy1**
I know that it's a small detail, but it could really confuse someone.

And fix spacing on line 112

### CODE REVIEW

**Comment by alallier** [comment]
While you are at it, see anything else? We might as well catch it with this one and tack it on

**Comment by yamboy1** [comment]
@alallier I'll have a look

**Comment by yamboy1** [comment]
Would it be a good idea to have a link to the license, rather than just saying `(MIT License)`. I would make the `(MIT License)` a link to your license file.

**Comment by alallier** [comment]
Sure go ahead and do that please

**Comment by yamboy1** [comment]
Ok

File: README.md [hunk 1]
```diff
@@ -198,21 +198,22 @@ Usage: reload [options]
...
-  -h, --help                        Output usage information
-  -V, --version                     Output the version number
-  -b, --browser                     Open in the browser automatically.
-  -n, --hostname                    If -b flag is being used, this allows for custom hostnames. Defaults to localhost.
-  -d, --dir [dir]                   The directory to serve up. Defaults to current dir.
-  -e, --exts [extensions]           Extensions separated by commas or pipes. Defaults to html,js,css.
-  -p, --port [port]                 The port to bind to. Can be set with PORT env variable as well. Defaults to 8080
-  -s, --start-page [start-page]     Specify a start page. Defaults to index.html.
-  -v, --verbose                     Turns on logging on the server and client side. Defaults to false.
+  -h, --help                     output usage information
```
Comment (alallier) [inline README.md]: What is the diff here?

Comment (alallier) [inline README.md]: Lets try and keep this to just any problems in docs you see like the wording problem. We can spice up and add things at a later time, as I have more doc changes coming later anyways

Comment (yamboy1) [inline README.md]: The main difference was adding `[verbose]` to the line 209, just so that it looked exactly like the output from `reload -h`

Comment (alallier) [inline README.md]: Okay that sounds good, why the diff on 201 to 208?

Comment (yamboy1) [inline README.md]: There isn't any difference. It might be because I am on linux and it is changing the line endings?

Comment (yamboy1) [inline README.md]: I just copied everything over from my terminal

**Comment by yamboy1** [comment]
In the [Table of Reload Parameters](https://github.com/alallier/reload#table-of-reload-parameters), is it really necessary to call the `opt` optional in the description when there is an optional checkbox next to it? 

Basically, can i remove `optional` from the description of `opt` because it seems superfluous.

**Comment by yamboy1** [comment]
And do we need curly braces around the `type` for the [Parameter tables](https://github.com/alallier/reload#parameters)?

File: README.md [hunk 2]
```diff
@@ -198,21 +198,22 @@ Usage: reload [options]
...
+  -s, --start-page [start-page]  Specify a start page. Defaults to index.html
+  -v, --verbose [verbose]        Turning on logging on the server and client side. Defaults to false
+
 ```
 
 License
 ---
 
-(MIT License)
+[(MIT License)](https://github.com/alallier/reload/blob/master/LICENSE)
```
Comment (alallier) [inline README.md]: Anyway to make this relative. Kind of like how I did here in the CHANGELOG `Please refer to the full API in the [README](README.md#api-for-express).` Should be able to reference the file without the URL

**Comment by alallier** [comment]
I like how the optional is the description it makes it really clear that it is optional and doesn't hurt to have it in there. The clearer we can be the less questions and problems our users will run into.

~~We don't necessarily need the brackets around the types, but it kinda feels right especially since in JSDocs the type is wrapped in brackets~~ Actually yeah you can remove the brackets and it will give consistency because I didn't do brackets in the return table.

File: README.md (hunk 1 above)
Comment (alallier) [inline README.md]: I see it's the spacing between the first and second columns that is different. This is good though, I like this because it is an exact copy of what `reload -h` provides.

File: README.md (hunk 2 above)
Comment (yamboy1) [inline README.md]: Fixed. See commit https://github.com/alallier/reload/pull/123/commits/6bacf6f64132b6232887de5e4ed91bb20227b4f8

Comment (yamboy1) [inline README.md]: Should I make the link to the expressSampleApp a relative link as well . Or if not, at least change the link from `jprichardson` to `alallier`.

Comment (alallier) [inline README.md]: Nice catch, yeah let's doing something with that. I don't know if you can make it relative? Try it out if not, the update from `jprichardson` to `alallier` would be great

File: README.md [hunk 3]
```diff
@@ -70,7 +70,7 @@ Reload can be used in conjunction with tools that allow for automatically restar
 ### Express Example
 
 **`server.js`:**
-```javascript
+```js
```
Comment (alallier) [inline README.md]: This is unnecessary diff. Let's leave this how it was.

**Comment by alallier** [comment]
Lets get that [link](https://github.com/alallier/reload/pull/123#discussion_r126552902) updated and get this merged in.

File: README.md (hunk 3 above)
Comment (yamboy1) [inline README.md]: Changed it back ab7340b

**Comment by yamboy1** [comment]
Changed the link to relative 3140125

File: README.md (hunk 3 above)
Comment (yamboy1) [inline README.md]: Change `javscript` to `javascript` 6fca6fe

**Comment by alallier** [comment]
Relative link is really cool. Looks good for merge. Planning to get this in with #128 and #125.

**Comment by alallier** [comment]
Thanks again!
